src/components/PathwayCourse/index.js

```diff
@@ -214,6 +221,35 @@ function PathwayCourse() {
                     <Typography variant="body1">
                       {pathwayCourseData.description}
                     </Typography>
+
+                    <ExternalLink
+                      style={{
+                        textDecoration: "none",
+                      }}
+                      href={"https://youtu.be/DDFvJmC3J5M"}
+                    >
+                      <Typography
+                        style={{ display: "flex" }}
+                        mt={2}
+                        align="start"
+                        variant="body2"
+                      >
+                        <img
+                          src={require("./asset/ComputerScreen.svg")}
+                          alt="MonitorScreen Img"
+                        />
+                        <section
+                          className={classes.link}
+                          onClick={handleVideo}
+                          style={{
+                            cursor: "pointer",
+                          }}
+                        >
+                          {"  "} &nbsp; &nbsp;
+                          <b>What's it all about?</b>
```

Comment:
Avoid using `<b>` if the only purpose is to make the text bold.  You can use CSS for this.  See https://developer.mozilla.org/en-US/docs/Web/HTML/Element/b for when you should use `<b>`.